src/components/organisms/proposal-summary.tsx

```diff
@@ -114,6 +114,7 @@ function renderMessage(msg: any, groupPolicyAddress: string) {
     case '/cosmos.bank.v1beta1.MsgSend':
       return (
         <SendReview
+          key={groupPolicyAddress}
```

Comment:
it made a linter warning disappear about not having a key in a list. i just lazily tried this out to see if would make the warning disappear and it seemed to work

Reply:
then items should have different keys because if you have a proposal with multiple msgs then you'll end up with another warning telling you: "Encountered two children with the same key" and that keys should be unique.

Reply:
i switched this so that we just use an index from the `.map` call instead:
1e8beeb